tasks/task01/Narcissistic.py

```diff
@@ -0,0 +1,13 @@
+for i in range(1,1001):
```

Comment:
the solution is correct. yet, several minor comments
1. it is better to add whitespace before and after operators. e.g. `x = i`, `x = x // 10`, `x == y`
2. `print` prints a new line by default, so there is no need to specify `end='\n'` argument.

Reply:
I fixed it